common-npm-packages/docker-common/registryauthenticationprovider/registryauthenticationtoken.ts

```diff
@@ -41,14 +41,14 @@ export default class RegistryServerAuthenticationToken {
     }
 
     public getDockerConfig(): string {
-        var authenticationToken = new Buffer(this.username + ":" + this.password).toString('base64')
+        var authenticationToken = Buffer.from(this.username + ":" + this.password).toString('base64')
```

Comment:
Can you explain this change please?

Reply:
https://stackoverflow.com/questions/52165333/deprecationwarning-buffer-is-deprecated-due-to-security-and-usability-issues